src/__init__.py

```diff
@@ -331,13 +336,33 @@ def add_filler_items(self, item_pool, traps):
             trap_count = extras * trap_percent // 100
             filler_count = extras - trap_count
 
-            for i in range(0, trap_count):
+            for _ in range(0, trap_count):
                 extra_item = self.create_item(self.random.choice(traps))
                 item_pool.append(extra_item)
 
-            for i in range(0, filler_count):
+            for _ in range(0, filler_count):
                 extra_item = self.create_item(filler_item_name)
                 item_pool.append(extra_item)
+        elif extras < 0:
+            logging.warning(f"{self.game} has more items than locations. {abs(extras)} useless items will be removed at random.")
```

Comment:
I think calling the items useless when useful is included could be misleading. We should change this to "non-progression".